test/resources/systemd.resource

```diff
@@ -56,6 +56,8 @@ Systemctl    # robocop: disable=too-long-keyword
     END
     Should Be Equal As Integers    0    ${rc}
 
+    IF    "${verb}" == "enable"    RETURN
```

Comment:
The docstring for the keyword should be updated if it supports "enable" as a verb.